tests/unit/test_collect_interface_tests.py

```diff
@@ -65,19 +69,17 @@ def test_registered_test_cases_cache(tmp_path, role, event, input_state, intf_na
     pth.write_text(
         dedent(
             f"""
-from interface_tester.interface_test import interface_test_case, Role
+from interface_tester.interface_test import register_test_case, Role
 from scenario import State
 
-@interface_test_case(
+register_test_case(
```

Comment:
I'm still somewhat troubled by the user experience of this. Can't we aim for tests to look like:

```python
from interface_tester import InterfaceTester

class MyInterfaceTest(InterfaceTester):
    def test_1(self):
        arrange()

        act()

        assert()
```

Reply:
I wish, but I don't see how to make that look natural. The arrange part is spread out between this repo and the charms, the act part is always the same and should not be customizable, so it's best left implicit, and in 99% of the cases, the same is true of the assert part.
Should we set up a call to brainstorm?